Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,125 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //declare my variables
+            string firstName;
+            string lastName;
+            float age;
+            float weightInKg;
+            float height;
+
+            //ask for first input of name
+            // then send call method to make sure it fits name paramter of only letters and assign to name variable
+            Console.WriteLine("Please enter your first name.");
+            firstName = CheckStringForLettersOnly();
+
+            // same as first name but for last name
+            Console.WriteLine("Please enter your last name.");
+            lastName = CheckStringForLettersOnly();
+
+
+            // same as name inputs but for age
+            //call age method and check to make sure it fits parameter of numbers only.
+            Console.WriteLine("Please enter your age.");
+            //I went with float for age because maybe someone likes to put percent of new age such as 7.5 years old...
+            age = float.Parse(CheckStringForDigitsOnly());
+
+            //same as age
+            Console.WriteLine("Please enter your weight in kg.");
+            //weightInKg = float.Parse(CheckStringForDigitsOnly(weightInKg.ToString()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4837469)